index.bs

```diff
@@ -1003,6 +1003,16 @@ must send a [=presentation-connection-open-response=] message and
 [=presentation-change-event=] messages when required.
 
 
+Representation Of Time {#time-representation}
+======================
+
+The [[#remote-playback-protocol]] and the [[#streaming-protocol]] represent
+points of time and durations in terms of a [=time scale=].  A <dfn>time
+scale</dfn> is a common denominator for time values that allows values which
+work on different time scales to be expressed as rational numbers without loss
+of precision.  The [=time scale=] is represented in hertz, such as 90000 for
+90000 Hz, a common time scale for video.
```

Comment:
Minor suggestion as it seems a bit weird to use "time scale" in the definition of "time scale". In practice, I think that part can be dropped without loss of meaning:

```suggestion
scale</dfn> is a common denominator for time values that allows values to be
expressed as rational numbers without loss of precision.  The [=time scale=] is
represented in hertz, such as 90000 for 90000 Hz, a common time scale for
video.
```